Header header
float32 x
float32 y
float32 z
float32 steering_delta
float32 heading_psi
float32 lean_phi